pkg/provider/kubernetes/ingress/kubernetes.go

```diff
@@ -221,17 +221,17 @@ func (p *Provider) loadConfigurationFromIngresses(ctx context.Context, client Cl
 			log.FromContext(ctx).Errorf("Error configuring TLS: %v", err)
 		}
 
-		if len(ingress.Spec.Rules) == 0 && ingress.Spec.Backend != nil {
+		if len(ingress.Spec.Rules) == 0 && (ingress.Spec.DefaultBackend != &networkingv1.IngressBackend{}) {
```

Comment:
We should keep the nil test here, since it's optional, we don't want to proceed further if the default backend is nil

Reply:
done :)